src/mame/drivers/mc10.cpp

```diff
@@ -26,123 +24,102 @@
 
 #include "formats/coco_cas.h"
 
-
-namespace
-{
```

Comment:
There should be no reason to remove this – src/devices shouldn’t be requiring symbols from src/mame anyway.